graphql/resolvers.go

```diff
@@ -1837,6 +1838,56 @@ func (r *taskResolver) LatestExecution(ctx context.Context, obj *restModel.APITa
 	return task.GetLatestExecution(*obj.Id)
 }
 
+func (r *queryResolver) SearchReturnInfo(ctx context.Context, taskId string, exec string) (*thirdparty.SearchReturnInfo, error) {
```

Comment:
The SearchReturnInfo resolver should call into a util function that can be used in old UI's search function to decrease the overall code we have to maintain until the old ui is completely removed. The util function can have the same parameters and and return values and logic as the resolver. These lines will be replaced in the service code with the util function: https://github.com/evergreen-ci/evergreen/blob/e8a4f20b335fe6efbfe1a0ec9086785003067445/service/ui_plugin_build_baron.go#L173-L205

Reply:
good idea, thanks!

Reply:
done